src/Functions/catboostEvaluate.cpp

```diff
@@ -0,0 +1,182 @@
+#include <Functions/FunctionHelpers.h>
+#include <Functions/FunctionFactory.h>
+
+#include <BridgeHelper/CatBoostLibraryBridgeHelper.h>
+#include <BridgeHelper/IBridgeHelper.h>
+#include <Columns/ColumnNullable.h>
+#include <Columns/ColumnString.h>
+#include <Columns/ColumnTuple.h>
+#include <Columns/ColumnsNumber.h>
+#include <Common/assert_cast.h>
+#include <DataTypes/DataTypeNullable.h>
+#include <DataTypes/DataTypeTuple.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <Functions/IFunction.h>
+#include <Interpreters/Context.h>
+#include <Interpreters/Context_fwd.h>
+
+
+namespace DB
```

Comment:
The code in this file is mostly copypasta of https://clickhouse.com/codebrowser/ClickHouse/src/Functions/modelEvaluate.cpp.html